distributor-api-v1/use-cases/on-session-payment-card-authorization.md

```diff
@@ -0,0 +1,28 @@
+# On session payment card authorization
+
+Whenever a user create reservation there is an option to let the guest insert a payment card which will be stored in Mews.
+You can read more about how to support payment cards in booking engine application in our [use-case](./how-to-support-payment-cards-in-booking-engine-application.md).
+With PSD2 directive it might be handy for the enterprise to authorize the card for smooth payments in the future. This can minimize
+the risk of a future payment being rejected.
+
+## Creating a reservation group with verified card
+
+First step of the entire workflow is to [create a reservation group via our API](../operations.md#create-reservation-group) with [payment card data](./how-to-support-payment-cards-in-booking-engine-application.md).
+The response contains a field `PaymentCardId` which we will use to check whether the card needs authorization via [requesting details of the card](../operations.md#get-payment-cards) and validating the [AuthorizationState](../operations.md#authorization-state) of the card is `Authorizable`.
```

Comment:
PaymentCardId is not saying anything about authorization for the card. AuthorizationState is saying if the CC has to be authorized.

Reply:
removed the mentioned `PaymentCardId` section. I will leave it only in the step-by-step workflow section